packages/kompose.yaml

```diff
@@ -0,0 +1,6 @@
+description: 'A conversion tool for all things compose (namely Docker Compose) to container orchestrators (Kubernetes or OpenShift)'
+webPage: 'http://kompose.io/'
```

Comment:
Is there a reason to prefer the non SSL version of this webpage?

Reply:
Hmm. Not really. I'll fix.